docker/Dockerfile.linux.arm

```
@@ -0,0 +1,10 @@
+FROM alpine:3.6 as alpine
```

```suggestion
FROM alpine:3.20 as alpine
```

mentioned all the occurrences of alpine and go